File: map/file.cpp
#include<iostream>
#include<map>
#include<string>
using namespace std;

int main()
{
	map<string,int> hashmap;
	hashmap["name"]=22;
	hashmap["age"]=33;
	cout<<hashmap["name"]<<endl;
	if (hashmap["year"]==0){
		cout<<-1<<endl;
	}
	else{
		cout<<hashmap["year"]<<endl;
	}
	hashmap.erase("age");

	for(auto itr=hashmap.begin();itr!=hashmap.end();itr++){
		cout<<itr->first<<" "<<itr->second<<endl;
	}

	return 0;
}

File: map/File.java
import java.util.Map;
import java.util.HashMap;
public class File{
	public static void main(String[] args){
		Map<String,Integer> map=new HashMap<>();
		map.put("name",3);
		map.put("age",4);
		System.out.println(map);
		System.out.println(map.getOrDefault("year",44));
		System.out.println(map.get("age"));
		map.remove("age");
		System.out.println(map);

	}
}
